src/hypertrace/agent/instrumentation/requests/__init__.py

```diff
@@ -37,9 +30,8 @@ def get_active_span_for_call(span, response) -> None:
 
         if span.is_recording():
             logger.debug('Span is recording.')
-            request_headers = [(k, v)
-                               for k, v in response.request.headers.items()] # pylint: disable=R1721
-            response_headers = [(k, v) for k, v in response.headers.items()] #pylint: disable=R1721
+            request_headers = response.request.headers
```

Comment:
Since we are not doing any conversion, do we still need to declare these variables?